Sources/PointFree/Episode/Show.swift

```diff
@@ -253,15 +233,15 @@ private let tocChapterLinkView = View<(title: String, timestamp: Int, active: Bo
       [
         div([`class`([Class.hide(.mobile)])], [
           a(
-            timestampLinkAttributes(timestamp: timestamp, useAnchors: true) +
```

Comment:
Removed this argument and am always linking directly to the video timestamp now.